static/app/views/starfish/views/appStartup/screenSummary/eventSamples.tsx

```diff
@@ -34,12 +36,16 @@ export function EventSamples({
   release,
   sortKey,
   showDeviceClassSelector,
+  footerAlignedPagination,
 }: Props) {
   const location = useLocation();
   const {selection} = usePageFilters();
   const {primaryRelease} = useReleaseSelection();
   const cursor = decodeScalar(location.query?.[cursorName]);
 
+  const deviceClass = decodeScalar(location.query[SpanMetricsField.DEVICE_CLASS]) ?? '';
+  const startType = decodeScalar(location.query[SpanMetricsField.APP_START_TYPE]) ?? '';
+
```

Comment:
You could make this a hook since it gets reused in other files

Reply:
Going to leave it this way for now because I have a TODO to clean up how we apply these "additional filters" everywhere. e.g. we also have a platform selector coming down the pipeline so I'm thinking of creating some kind of filtering context that provides the applicable filters